kubeflow/contrib/kserve/patches/namespace.yaml

```diff
@@ -1,6 +0,0 @@
-# Remove namespace resource as namespace will already exist.
-$patch: delete
-apiVersion: v1
-kind: Namespace
-metadata: # kpt-merge: /kserve
-  name: kserve
```

Comment:
We should remove kserve namespace, right? Or is upstream handling this case?

Reply:
I don't see this namespace. The patch is not used anywhere either. We can keep it if you have a strong opinion.

I could find the namespace after deployment:


**this commit**:
```bash
> kubectl get namespace

NAME              STATUS   AGE
ablai             Active   19h
asm-system        Active   23h
cert-manager      Active   23h
default           Active   23h
istio-system      Active   23h
knative-serving   Active   23h
kserve-test       Active   18h
kube-node-lease   Active   23h
kube-public       Active   23h
kube-system       Active   23h
kubeflow          Active   23h
```

**1.6.1**:
```bash
> kubectl get namespace

NAME              STATUS   AGE
ablai             Active   8d
asm-system        Active   9d
cert-manager      Active   9d
default           Active   9d
istio-system      Active   9d
knative-serving   Active   9d
kserve-test       Active   24h
kube-node-lease   Active   9d
kube-public       Active   9d
kube-system       Active   9d
kubeflow          Active   9d
```

The `kserve-test` and `ablai` namespaces were created during [kserve testing](https://kserve.github.io/website/master/get_started/first_isvc/#2-create-an-inferenceservice) and [e2d MNIST testing](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb).

Reply:
I think the intention of this file is to remove any pre-existing kserve deployment. Multiple kserve deployment in a single GKE cluster will likely cause problem.

Reply:
That makes sense for cases whith custom `kserve` installation. However, this patch has been commented out more [than year ago](https://github.com/GoogleCloudPlatform/kubeflow-distribution/pull/355) probably because it causes this error:
```
Error: no matches for Id ~G_v1_Namespace|~X|kserve; failed to find unique target for patch ~G_v1_Namespace|kserve
make: *** [Makefile:23: hydrate] Error 1
```

Should we keep it commented out and store the patch yaml (not sure is anyone will be affected if we remove the dead code)?